src/components/Reports/SalaryCalculator/staffTypeHelpers.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  AssignmentCategoryEnum,
+  AssignmentStatusEnum,
+  PeopleGroupSupportTypeEnum,
+  StaffInfo,
+  UserPersonTypeEnum,
+} from 'src/graphql/types.generated';
+
+/**
+ * Checks if staff member is a full-time RMO.
+ * Reference Status Code: SC
+ */
+export const isFullTimeRmo = (staffInfo: StaffInfo): boolean => {
+  return (
+    staffInfo.userPersonType === UserPersonTypeEnum.EmployeeStaff &&
+    staffInfo.peopleGroupSupportType ===
+      PeopleGroupSupportTypeEnum.SupportedRmo &&
+    staffInfo.assignmentStatus === AssignmentStatusEnum.ActivePayrollEligible &&
+    staffInfo.assignmentCategory === AssignmentCategoryEnum.FullTimeRegular
+  );
+};
+
+/**
+ * Checks if staff member is new staff.
+ * Reference Status Codes: SA, SS, SD
+ */
+export const isNewStaff = (staffInfo: StaffInfo): boolean => {
+  const isValidPersonType =
+    staffInfo.userPersonType === UserPersonTypeEnum.PendingStaff ||
+    staffInfo.userPersonType === UserPersonTypeEnum.EmployeeStaff;
+
+  const isValidSupportType =
+    staffInfo.peopleGroupSupportType ===
+      PeopleGroupSupportTypeEnum.SupportedRmo ||
+    staffInfo.peopleGroupSupportType ===
+      PeopleGroupSupportTypeEnum.SupportedNonRmo;
+
+  const isValidStatus =
+    staffInfo.assignmentStatus === AssignmentStatusEnum.InactiveNoPayroll ||
+    staffInfo.assignmentStatus === AssignmentStatusEnum.PendingNoPayroll ||
+    staffInfo.assignmentStatus ===
+      AssignmentStatusEnum.RaisingInitialPayrollEligible ||
+    staffInfo.assignmentStatus === AssignmentStatusEnum.RaisingInitialNoPayroll;
```

Comment:
Where does this fourth condition come from? I see that the other three are for SD, SA, and SS, respectively. Do we need to add SSS to the reference status codes at the top?